authority-portal-frontend/src/app/pages/catalog-page/property-grid/property-grid.component.html

```diff
@@ -7,6 +7,7 @@
   class="flex flex-row gap-2 overflow-hidden">
   <!-- Property Icon -->
   <mat-icon
+    class="text-gray-500"
```

Comment:
While I understand that we use gray icons in other AP components, I don't know if the change is good here. The whole grid looks a little too gray now? EDC UI also uses black icons.

It looks fine, but I liked the black icons more.